src/server/services/MusicresServices.js

```diff
@@ -39,17 +38,88 @@ class MusicresServices {
   }
 
   static async getAllMusicReserves(term) {
-    const termMedia = await MediaQuery.getMediaForTerm('musicreserves', term);
+    const termMedia = await MediaQuery.getMediaForTerm(term);
```

Comment:
Doesn't `getMediaForTerm` require two parameters? One for the collection, and one for the term?